### ISSUE TITLE
Align the order of chart bars with the order of categories in the filters

### ISSUE THREAD

**Issue by lmichael107** [CaRCC/rcd-nexus-portal#197]
For some "Chart Views:", the displayed chart bars are not logically ordered like categories are in the lists under "Filters:"

See attached, for Mission categories, where some are also missing (per #196).

![Screenshot 2024-08-26 at 2 51 07 PM](https://github.com/user-attachments/assets/1cc2f015-08d3-46d0-9124-eb2ba3ad7bc8)

**Comment by plschmitz** [comment]
Try mapping the mission2 to a numeric value, sort, and then map to the strings. Maybe this will work.

**Comment by plschmitz** [comment]
Fixed on current branch
